feature-libs/product-configurator/rulebased/components/price-summary/configurator-price-summary.component.html

```diff
@@ -1,30 +1,32 @@
 <ng-container *ngIf="configuration$ | async as configuration">
-  <div class="cx-price-summary-container">
-    <div class="cx-total-summary">
-      <div class="cx-summary-row cx-base-price">
-        <div class="cx-label">
-          {{ 'configurator.priceSummary.basePrice' | cxTranslate }}:
+  <ng-container *ngIf="configuration.pricingEnabled">
```

Comment:
Should `cxFeatureLevel` be added to HTML tree?

Reply:
No, because the HTML tree doesn't change (in case pricing is enabled)